src/main/java/com/redhat/rhjmc/containerjfr/net/web/WebServer.java

```diff
@@ -416,7 +420,7 @@ void handleAuthRequest(RoutingContext ctx) {
     }
 
     void handleClientUrlRequest(RoutingContext ctx) {
-        ctx.response().putHeader(HttpHeaders.CONTENT_TYPE, MIME_TYPE_JSON);
+
```

Comment:
Why was this removal needed?

Reply:
I'm afraid you're looking at an out-dated commit. I had this line removed by mistake. It was re-added later.